ui-speedspacechart/src/components/helpers/drawElements/curve.ts

```diff
@@ -9,10 +9,11 @@ import { clearCanvas, maxPositionValue, maxSpeedValue } from '../../utils';
 
 const { CURVE_MARGIN_TOP, CURVE_MARGIN_SIDES } = MARGINS;
 
-const drawSpecificCurve = (
+const traceCurve = (
   curveConfig: CurveConfig,
   canvasConfig: CanvasConfig,
-  specificSpeeds: LayerData<number>[]
+  specificSpeeds: LayerData<number>[],
+  closePath: boolean
```

Comment:
Even though it's in the same file, we recently decided to use an object when passing a boolean in a function param so it can be more explicit why we call i with `true` or `false`

```suggestion
  { closePath: boolean }
```

Reply:
Okay... Is it optimized during the JS transformation?
It's something that the code editor should handle.

Reply:
Fixed here: https://github.com/OpenRailAssociation/osrd-ui/pull/509/commits/0e65f5a6a56c0fa79afce0a8271343676bbb2945.

I'm really not a big fan of this convention. What about putting the boolean into a viriable instead of buildign a record?

Reply:
Ah sorry for the confusion, I wasn't very clear maybe. The goal was to use the function like this on in osrd for example : 
`export function secToHoursString(sec: number | null, { withSeconds = false } = {}): TimeString {...}`

and then use it like that when calling the function : 
`secToHoursString(durationInSecond, { withSeconds: true })`

Is it ok for you ?

Reply:
In my case I have one variable and I don't want a default value to it. I don't see how do you want me handle this.

> [!NOTE]
> Making the function arguments more explicit seems like a good idea, but it makes the function declaration less readable.